larray/core.py

```diff
@@ -5996,8 +6047,17 @@ def info(self):
         2 x 2
          nat [2]: 'BE' 'FO'
          sex [2]: 'M' 'F'
+        >>> mat1 = LArray(np.ones((2, 2)), [nat, sex], 'test matrix')
```

Comment:
we should implement title in all array creation functions, so that we could write something like this instead:
```python
mat1 = ones((nat, sex), 'test matrix')
```
This should be in anohter PR though.